### PR TITLE
New translations for !tracked and roll-up of other outstanding translation issues

### PR DESCRIPTION
**Pull request by bbdoc**
Added translations to !tracked as most text was harcoded and not going through the translator. Everything is now going through the translator.

Added translation for monsters shown in !tracked as you could add monsters to tracking in your local languages but they were still being shown in English when using !tracked.

Added French translations to fr.json for all existing translated items in !tracked + all new items added to translator.

Tested on my instance, running fine.

Poracle

### CODE REVIEW

File: src/lib/discord/commando/commands/tracked.js [hunk 1]
```diff
@@ -41,14 +41,16 @@ exports.run = async (client, msg, [args]) => {
...
-		await msg.reply(`Your alerts are currently ${human.enabled ? 'enabled' : 'disabled'}\n${locationText} You are currently set to receive alarms in ${human.area}`)
+		const alertResponseString1 = client.translator.translate('Your alerts are currently ').concat(client.translator.translate(human.enabled ? 'enabled' : 'disabled'))
+                const alertResponseString2 = client.translator.translate('You are currently set to receive alarms in ').concat(human.area)
+                await msg.reply(alertResponseString1.concat('\n', locationText, '\n', alertResponseString2, '\n\n'))
 
 		let message = ''
 		if (monsters.length) {
-			message = message.concat(client.translator.translate('\n\nYou\'re  tracking the following monsters:\n'))
-		} else message = message.concat(client.translator.translate('\n\nYou\'re not tracking any monsters'))
+			message = message.concat('\n\n', client.translator.translate('You\'re  tracking the following monsters:'),'\n')
```
Comment (jfberry) [inline src/lib/discord/commando/commands/tracked.js:52]: Worth fixing the double space here

File: src/lib/discord/commando/commands/tracked.js [hunk 2]
```diff
@@ -41,14 +41,16 @@ exports.run = async (client, msg, [args]) => {
...
+		} else message = message.concat('\n\n', client.translator.translate('You\'re not tracking any monster'))
```
Comment (jfberry) [inline src/lib/discord/commando/commands/tracked.js:53]: You have lost the plural 'tracking any monsters'

File: src/lib/discord/commando/commands/tracked.js [hunk 3]
```diff
@@ -59,17 +61,17 @@ exports.run = async (client, msg, [args]) => {
...
 				if (formName === undefined) formName = 'none'
 			}
 			let miniv = monster.min_iv
 			if (miniv === -1) miniv = 0
 			message = message.concat(`\n**${monsterName}** form: ${formName} ${monster.distance ? `, distance: ${monster.distance}m` : ''} iv: ${miniv}%-${monster.max_iv}% cp: ${monster.min_cp}-${monster.max_cp} level: ${monster.min_level}-${monster.max_level} stats: ${monster.atk}/${monster.def}/${monster.sta} - ${monster.max_atk}/${monster.max_def}/${monster.max_sta}, gender:${client.utilData.genders[monster.gender].emoji}`)
 		})
 		if (raids.length || eggs.length) {
-			message = message.concat(client.translator.translate('\n\nYou\'re tracking the following raids:\n'))
-		} else message = message.concat(client.translator.translate('\n\nYou\'re not tracking any raids'))
+			message = message.concat('\n\n', client.translator.translate('You\'re  tracking the following raids:'),'\n')
```
Comment (jfberry) [inline src/lib/discord/commando/commands/tracked.js:73]: extra space

File: src/lib/discord/commando/commands/tracked.js [hunk 4]
```diff
@@ -59,17 +61,17 @@ exports.run = async (client, msg, [args]) => {
...
+		} else message = message.concat('\n\n', client.translator.translate('You\'re not tracking any raid'))
```
Comment (jfberry) [inline src/lib/discord/commando/commands/tracked.js:74]: missing plural

File: src/lib/discord/commando/commands/tracked.js [hunk 5]
```diff
@@ -88,8 +90,8 @@ exports.run = async (client, msg, [args]) => {
 		})
 
 		if (quests.length) {
-			message = message.concat(client.translator.translate('\n\nYou\'re tracking the following quests:\n'))
-		} else message = message.concat(client.translator.translate('\n\nYou\'re not tracking any quests'))
+			message = message.concat('\n\n', client.translator.translate('You\'re  tracking the following quests:'),'\n')
```
Comment (jfberry) [inline src/lib/discord/commando/commands/tracked.js:93]: extra space

File: src/lib/discord/commando/commands/tracked.js [hunk 6]
```diff
@@ -88,8 +90,8 @@ exports.run = async (client, msg, [args]) => {
...
+		} else message = message.concat('\n\n', client.translator.translate('You\'re not tracking any quest'))
```
Comment (jfberry) [inline src/lib/discord/commando/commands/tracked.js:94]: missing plural

File: src/lib/discord/commando/commands/tracked.js [hunk 7]
```diff
@@ -101,8 +103,8 @@ exports.run = async (client, msg, [args]) => {
 		})
 
 		if (invasions.length) {
-			message = message.concat(client.translator.translate('\n\nYou\'re tracking the following invasions:\n'))
-		} else message = message.concat(client.translator.translate('\n\nYou\'re not tracking any invasions'))
+			message = message.concat('\n\n', client.translator.translate('You\'re  tracking the following invasions:'),'\n')
```
Comment (jfberry) [inline src/lib/discord/commando/commands/tracked.js:106]: extra space

File: src/lib/discord/commando/commands/tracked.js [hunk 8]
```diff
@@ -101,8 +103,8 @@ exports.run = async (client, msg, [args]) => {
...
+		} else message = message.concat('\n\n', client.translator.translate('You\'re not tracking any invasion'))
```
Comment (jfberry) [inline src/lib/discord/commando/commands/tracked.js:107]: missing plural

**Review by jfberry — CHANGES_REQUESTED** [review]
Superb piece of work, some minor typos (some were already in the source and some you have introduced) would make this a no-brainer

**Comment by bbdoc** [comment]
Thanks, that's corrected. I indeed thought that "not any" was taking a singular as there is logically 0, as would be the case in french ;-) but you are right, I learned something new today ;-) 

`not any is plural with plural count nouns and singular with non-count nouns`

File: src/lib/discord/commando/commands/tracked.js (hunk 5 above)
Comment (bbdoc) [reply src/lib/discord/commando/commands/tracked.js]: Done

**Comment by jfberry** [comment]
All changes look good -- ready for merge!

**Comment by jfberry** [comment]
Merged these changes by hand, thanks bbdoc!

**Pull request closed without merging**